doc/rtd/index.rst

```diff
@@ -44,7 +44,7 @@ Having trouble? We would like to help!
    :titlesonly:
    :caption: Getting Started
 
-   topics/tutorial.rst
+   topics/tutorials/lxd.rst
```

Comment:
I'm a bit confused by the doc structure here. Is the Qemu tutorial subordinate to the LXD one, or are they just different approaches to achieving the same end? If the latter,they should really be at the same level in the hierarchy. In that case, I'd say yes, absolutely we need a tutorials landing page (although I'm intending to make one as part of our Diataxification efforts).

Although then I would have to ask "do we need two tutorials on the same topic?". Is one of these methods more common, easier to use, or otherwise preferable in some way? If so, that should be what we use to get people started, and the other tutorial may be better as a how-to guide.

Reply:
Since developers' preferred virtualization "test environment" tooling varies from person to person (qemu, libvirt, lxd, virtualbox, vagrant, etc). I think it is desirable to provide multiple tutorials.

Additionally, Qemu is capable of running virtual machines on [BSD](https://wiki.qemu.org/Hosts/BSD) / [MacOS](https://wiki.qemu.org/Hosts/Mac), while [LXD does not](https://ubuntu.com/blog/lxd-client-on-windows-and-macos), which means that this tutorial may allow some users to test locally despite lacking LXD support.

I think a landing page makes sense, but with only 2 tutorials I wasn't sure the best way to go about that (a list with only 2 items seems a bit hollow).

Reply:
It's an interesting problem. Diataxis' end goal is a single end-to-end tutorial, although that may or may not be appropriate here. 

The key is going to be signposting, I think. If the methods really are equivalent but the difference is *who* might use each one, we need to include some context around that decision so that people can pick the right tutorial for their needs without having to skim each one first (saving time and effort for the user). E.g., someone who wants to take part in development (I'm guessing) has a different set of needs than a sysadmin. If each tutorial is better for a different purpose, we can (and probably should) state that explicitly to make it clear, or at least provide the brief explanation you just made as an introduction on the tutorial landing page (which we will need) to explain the two options.

Out of curiosity, is there a tool that would be ideal for non-dev/non-sysadmin types who want to try cloud-init?

Reply:
> is there a tool that would be ideal for non-dev/non-sysadmin types who want to try cloud-init?

Hmmm, Virtualbox probably most "cross-platform", but Qemu is more widely used on Linux. I don't have a good answer to this.

Reply:
After this PR is merged, I'll be creating Diataxis section landing pages, so I can easily link both tutorials from the Tutorial landing page (fixing the navigation issue). If it means we have two tutorials for a while, I'm totally OK with that. My personal preference would be to use the LXD one as a quick-start tutorial since it could easily fill that gap as-is. I completed it before np, so I think it works well for the subset of users who don't need introducing to the concepts. 

For the time being, I could simply include introductory text on the Tutorial landing page to direct users to the tutorial that they want to use? (quick start LXD vs detailed Qemu). It's a tidy option, and might end up working really well from the users' point of view. In this way we have two bases covered, and serve two different subsets of users with the Tutorial content.

Reply:
+1, sounds good to me!

Reply:
Thanks all for the reviews! I don't see any unaddressed issues[1]. I'll merge once tests pass, barring any last minute objections.

[1] Thanks @s-makin for offering to pick up the navigation with Diataxification!!